Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/view/task_create/TaskCreateFragment.java

```diff
@@ -135,8 +134,14 @@ private Long saveOrUpdate() {
 
     private Long addTask() {
         try {
+            Integer duration = null;
```

Comment:
The code in add and update is the same, I think it would be nice to move it to one method that both could use.

Reply:
Okay, I am thinking "private Integer getDuration()", would that be okay with you, or too confusing since it is not really a public getter?

Reply:
getDuration is good :)